### PR TITLE
Set up testing framework - RSpec

### PR DESCRIPTION
**Pull request by RhonGabriel**
# Update
Installed ruby version 2.4.3
Deleted comments
_______
# PT 
https://www.pivotaltracker.com/story/show/160582051
_______
# Gems
Added the following gem to `group :development, :test`
```
- gem 'factory_bot_rails'
- gem 'pry-byebug'
- gem 'pry-rails'
- gem 'rspec-rails'
- gem 'shoulda-matchers'
```
Executed Rspec generator

### CODE REVIEW

**Review by tochman — CHANGES_REQUESTED** [review]
These commits are also part of the next PR. Why is that?

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--require spec_helper
```
Comment (tochman) [inline .rspec]: require `rails_helper` here

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.3'
+ruby '2.5.1'
```
Comment (tochman) [inline Gemfile]: Never change this.

File: Gemfile [hunk 3]
```diff
@@ -16,6 +16,16 @@ gem 'sass-rails', '~> 5.0'
 gem 'turbolinks'
 gem 'uglifier', '>= 1.3.0'
 
+group :development, :test do
+  gem 'cucumber-rails', require: false
```
Comment (tochman) [inline Gemfile]: ???

File: Gemfile [hunk 4]
```diff
@@ -16,6 +16,16 @@ gem 'sass-rails', '~> 5.0'
...
+  gem 'database_cleaner'
```
Comment (tochman) [inline Gemfile]: ???

File: features/support/env.rb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+#   Before('not @no-txn', 'not @selenium', 'not @culerity', 'not @celerity', 'not @javascript') do
+#     DatabaseCleaner.strategy = :transaction
+#   end
+#
+
+# Possible values are :truncation and :transaction
+# The :transaction strategy is faster, but might give you threading problems.
+# See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
+Cucumber::Rails::Database.javascript_strategy = :truncation
+
```
Comment (tochman) [inline features/support/env.rb]: Out of scope. Relates to cucumber

File: spec/rails_helper.rb [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  config.filter_rails_from_backtrace!
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
+end
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails 
+  end
+end
```
Comment (tochman) [inline spec/rails_helper.rb:30]: Please remove comments in file

File: spec/spec_helper.rb [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (tochman) [inline spec/spec_helper.rb]: Please remove comments

**Review by tochman — CHANGES_REQUESTED** [review]

**Comment by RhonGabriel** [comment]
Fixed issue with ruby version and deleted comments from spec folder (rails_helper.rb and spec_helper.rb)

**Review by tochman — APPROVED** [review]
Looks good to me. Good job.

**Pull request merged**